www/src/helpers/authentication.js

```diff
@@ -10,4 +10,17 @@ export function fetchToken() {
 
 export function setToken(token) {
   localStorage.setItem(AUTH_TOKEN, token)
-}
\ No newline at end of file
+}
+
+export function setPreviousUserData(userData) {
+  localStorage.setItem(AUTH_PREVIOUS_USER_DATA, JSON.stringify(userData))
```

Comment:
Does this have to be stringified?  I feel like local storage should be able to just store the raw user.  Alternatively, we could just have this store only the jwt as that's all that's necessary.

Reply:
Yes localStorage can only store string key/value pairs.

Reply:
in that case, perhaps we should just store the jwt here to avoid the annoying json serialization logic, esp since I believe it's all that's need to restore the previous user.

Reply:
Oh wait, I see, you're trying to keep name and avatar and all that, that makes sense